android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.2-all.zip
```

Comment:
The migration guide mentions 8.12, but this one uses 8.10.2, any reason for it?

Reply:
There was a mistake in the link I sent because that one had version 0.78.0, which was my first attempt. But this is the right one https://react-native-community.github.io/upgrade-helper/?from=0.73.7&to=0.76.7, we moved to 0.76.7 due to some incompatibilities from mapbox from version 0.77.0 and up.